api/v1/config.js

```diff
@@ -192,6 +192,13 @@ module.exports = {
       description:
         'Eden has bugs too. :( See what issues we have and chime in if you found a bug!',
     },
+    {
+      url: 'https://github.com/EdenServer/eden-web/issues',
+      image: 'https://static.ffxiah.com/images/icon/17001.png',
```

Comment:
Maybe we can have a different icon for website bugs?

Reply:
Is linking to images that are not self-hosted a bad practice?  Or did we obtain permission?  Whichever the case, how about: https://static.ffxiah.com/images/icon/17005.png (Lufaise Fly)